tests/runner.py

```diff
@@ -181,12 +195,17 @@ def hide_does_not_affect_capturing(self):
     class pseudo_terminals:
         def return_value_indicates_whether_pty_was_used(self):
             eq_(run("true").pty, False)
-            eq_(run("true", pty=True).pty, True)
+            if not WINDOWS:
+                eq_(run("true", pty=True).pty, True)
 
         def pty_defaults_to_off(self):
             eq_(run("true").pty, False)
 
         def complex_nesting_doesnt_break(self):
+            # Not sure how to make this work on Windows
+            if WINDOWS:
+                skip()
```

Comment:
Might make sense to turn this into a decorator like e.g. `@posix_only` or `@no_windows`? since it's used a bunch.

Could potentially also use Nose's built in tagging stuff and then run the tests with a "skip windows tag" option, though I've never used that nor do I know offhand whether spec would break it as-is.